### PR TITLE
bugfix:fix NPE and cluster key about apollo

### LINKED ISSUE #3677: 1.4.1升级到1.4.2后,apollo config 报错
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
seata server 1.4.1 -> 1.4.2, registry config from apollo, not change codes or configs.
1.4.1 works well, but 1.4.2 cannot run

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
SLF4J: A number (17) of logging calls during the initialization phase have been intercepted and are
SLF4J: now being replayed. These are subject to the filtering rules of the underlying logging system.
SLF4J: See also http://www.slf4j.org/codes.html#replay
02:10:00.280  INFO --- [                     main] io.seata.server.Server                   : The server is running in container.
02:10:00.380  INFO --- [                     main] io.seata.config.FileConfiguration        : The file name of the operation is file:/root/seata-config/registry
02:10:00.382  INFO --- [                     main] io.seata.config.FileConfiguration        : The configuration file used is /root/seata-config/registry.conf
Exception in thread "main" java.lang.NullPointerException
        at java.util.Hashtable.put(Hashtable.java:460)
        at java.util.Properties.setProperty(Properties.java:166)
        at java.lang.System.setProperty(System.java:798)
        at io.seata.config.apollo.ApolloConfiguration.readyApolloConfig(ApolloConfiguration.java:188)
        at io.seata.config.apollo.ApolloConfiguration.<init>(ApolloConfiguration.java:77)
        at io.seata.config.apollo.ApolloConfiguration.getInstance(ApolloConfiguration.java:111)
        at io.seata.config.apollo.ApolloConfigurationProvider.provide(ApolloConfigurationProvider.java:29)
        at io.seata.config.ConfigurationFactory.buildConfiguration(ConfigurationFactory.java:131)
        at io.seata.config.ConfigurationFactory.getInstance(ConfigurationFactory.java:94)
        at io.seata.server.ParameterParser.init(ParameterParser.java:84)
        at io.seata.server.ParameterParser.<init>(ParameterParser.java:60)
        at io.seata.server.Server.main(Server.java:63)

```

### LINKED ISSUE #3684: 1.4.2版本代码bug
版本：seata 1.4.2版本
问题：apollo配置中心cluster代码问题，常量APOLLO_CLUSTER值应该叫cluster，还有apolloConfigService应该和apolloMeta是同一台服务器，理论上应该是同一个地址
类路径：io.seata.config.apollo.ApolloConfiguration

### PR DESCRIPTION
**Pull request by lightClouds917**
### Ⅰ. Describe what this PR did
fix:NPE about apollo setProperty

### Ⅱ. Does this pull request fix one issue?

fixes #3677 
fixes #3684

### CODE REVIEW

File: config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java [hunk 1]
```diff
@@ -173,22 +173,34 @@ public void removeConfigListener(String dataId, ConfigurationChangeListener list
...
         if (!properties.containsKey(PROP_APP_ID)) {
-            System.setProperty(PROP_APP_ID, FILE_CONFIG.getConfig(getApolloAppIdFileKey()));
+            String appId = FILE_CONFIG.getConfig(getApolloAppIdFileKey());
+            if (StringUtils.isNotBlank(appId)) {
+                System.setProperty(PROP_APP_ID, appId);
+            }
         }
         if (!properties.containsKey(PROP_APOLLO_META)) {
-            System.setProperty(PROP_APOLLO_META, FILE_CONFIG.getConfig(getApolloMetaFileKey()));
+            String apolloMeta = FILE_CONFIG.getConfig(getApolloMetaFileKey());
```
Comment (a364176773) [inline config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java:182]: apollo-meta 和apollo.configService 作用一致，应该提前判断，如果二者都为空，要抛出异常提示用户配置中心初始化失败

Comment (lightClouds917) [reply config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java:182]: finished

**Review by a364176773 — APPROVED** [review]
LGTM

**Comment by ppj19891020** [comment]
LGTM

**Comment by a364176773** [comment]
@xingfudeshi PTAL

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**